src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -413,6 +415,20 @@ export default {
         }
     },
     methods: {
+        expand() {
+            const view = {
```

Comment:
I think the view passed in should just be this view, no? We shouldn't need to create a new one. It can be passed down via `inject`?